actors/market/src/lib.rs

```diff
@@ -635,11 +637,17 @@ impl Actor {
 
                 sectors_deals
                     .push((sector.sector_number, SectorDealIDs { deals: sector.deal_ids.clone() }));
-                activations.push(SectorDealActivation {
-                    nonverified_deal_space,
-                    verified_infos,
-                    unsealed_cid: data_commitment,
-                });
+                activations.push(SectorDealActivation { activated, unsealed_cid: data_commitment });
+
+                for (deal_id, proposal) in sector.deal_ids.iter().zip(&validated_proposals) {
```

Comment:
Any reason not to combine this with the previous for loop?

Reply:
Just makes it easy to see that events are only emitted after everything else has been handled. For example, see [here](https://github.com/filecoin-project/builtin-actors/blob/7e0abcacc4ad8c1d1ff4ef404f9f173380388acf/actors/market/src/lib.rs#L631).

Reply:
Sg.